tardis/adapters/batchsystems/slurm.py

```diff
@@ -131,7 +133,7 @@ async def drain_machine(self, drone_uuid: str) -> None:
 
         cmd = f"scontrol update NodeName={machine} State=DRAIN Reason='COBalD/TARDIS'"
 
-        return await async_run_command(cmd)
+        return await self._executor.run_command(cmd)
```

Comment:
According to the docstring this function should return `None`. It was returning `stdout` before, the return value of `async_run_command`. I would suggest to you remove the `return` in front of `await self._executor.run_command` to make it consistent to the documentation.